tests/Doctrine/Tests/ORM/Query/SelectSqlGenerationTest.php

```diff
@@ -31,9 +31,13 @@
 use Doctrine\Tests\OrmTestCase;
 use Exception;
 
+use function class_exists;
 use function get_class;
 use function sprintf;
 
+// DBAL 2 compatibility
+class_exists('Doctrine\DBAL\Platforms\MySqlPlatform');
```

Comment:
That's a tricky one. The class name was changed from `MySqlPlatform` to `MySQLPlatform`. Class names are case-insensitive in PHP, so just using the new name would theoretically work. However, on a case-sensitive filesystem, autoloading would fail if the case does not match. This is why I'm triggering the autoloader with old and new class name.